### PR TITLE
Miscellaneous improvements

### PR DESCRIPTION
**Pull request by bitsofbits**
* Support regular colorbars for bivariate rasters
* Have Bivariate rasters correctly support discretized colormaps using BoundaryNorm
* Add tile_downloader for downloading remote map tiles
* Add examples
* Render example notebook as MD so it's viewable on github.
* Misc bug fixes

### CODE REVIEW

**Review by jennvanosdel — COMMENTED** [review]
Just small changes. The biggest thing is an issue with `psm.style.dark` and its background and consistent use in `Examples.md`. Super excited to see the .md be used and be stored directly in the repo! Also really cool stuff on the bivariate maps and their colorbars!

File: pyseas/doc/Examples.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
 # # Examples of Plotting with *pyseas*
 
+# +
+# pip install numba
```
[the remark this answers is not in the document]
Comment (bitsofbits) [reply pyseas/doc/Examples.py:5]: Yep!

File: pyseas/doc/Examples.py [hunk 2]
```diff
@@ -36,10 +40,20 @@
 # light and dark styles, which are activated using `pyseas.context`.
 
 with psm.context(psm.styles.dark):
```
[the remark this answers is not in the document]
Comment (bitsofbits) [reply pyseas/doc/Examples.py:42]: Yes, I've seen these. On my agenda for next week(?) is putting together an updated radenv that doesn't issue warnings.

File: pyseas/doc/Examples.py [hunk 3]
```diff
@@ -36,10 +40,20 @@
...
-    fig = plt.figure(figsize=(18, 6))
+    fig = plt.figure(figsize=(16, 9))
     psm.create_map(projection="regional.european_union")
     psm.add_land()
 
+# It is often convenient to set either the horizontal or vertical extent and have
+# the map fill the rest of the figure. This can done using `set_lon_extent` or
+# `set_lat_extent`.
+
+with psm.context(psm.styles.dark):
```
[the remark this answers is not in the document]
Comment (bitsofbits) [reply pyseas/doc/Examples.py:51]: Part 2 I addressed by setting a default figure facecolor to  `background.color`. This means the creation of the figure
has to happen inside the context, so I've adjust all the examples to reflect this.

Part 1: was a result of some bad settings in props.json.

And fixing these exposed an _reload bug, so had to fix that too.

File: pyseas/maps/colorbar.py [hunk 4]
```diff
@@ -34,33 +36,51 @@ def add_colorbar(
...
         Vertical alignment of center of text relative to colorbar.
     right_edge : float, optional
         Location of right edge of colorbar in Axes coords.
+    **kwargs: passed to colorbar
 
     Returns
     -------
     Colorbar
     """
+    not center and label
```
[the remark this answers is not in the document]
Comment (bitsofbits) [reply pyseas/maps/colorbar.py:52]: Yes.

I don't think you can do much with this kind of boolean, although you could do all sorts of nonsense with the binary version `~center & label`. If you were a monster.

**Pull request merged**